### DISCUSSION TITLE
[@types/google-apps-script] Support new Document Tab API

### DISCUSSION THREAD

**Discussion by flavianh** [DefinitelyTyped/DefinitelyTyped#70386] [Issues with a @types package]
Since [August 15 2024](https://developers.google.com/apps-script/docs/release-notes#August_15_2024) the Apps Script Document API exposes a tab API (unavailable in the UI at this time).
Changes are backwards-compatible for documents with a single tab.
New objects and methods are available here: https://developers.google.com/apps-script/guides/docs/tabs
I think it would be great for DefinitelyTyped to support this change as soon as possible.

[a comment by an automated account is not shown]

**Comment by mtgto** [comment]
Thanks for the report. There doesn't seem to be a definition for tabs, as you say.
I hope you, or someone else who sees this discussion, will create a pull request.

**Reply by flavianh** [reply]
Well, they link to a definition here: https://developers.google.com/apps-script/reference/document/tab

I won't be able to provide a PR but I hope someone finds the motivation to

**Comment by tomasdev** [comment]
Definitions added in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71856
